### PR TITLE
Kyle/v4 master merge

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/1047

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/organisms/BasketOverview/hooks/useBasketOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    const minStartDate = basket.basketInfo?.dateCriteria?.minStartDate;
+    if (minStartDate) {
+      _overview.minStartDate = minStartDate.toISOString();
+    }
+
+    const startDateWindow = basket.basketInfo?.dateCriteria?.startDateWindow;
+    if (startDateWindow) {
+      _overview.startDateWindow = formatDuration(
+        startDateWindow.seconds.toNumber(),
```
Comment (mhagel) [inline web-registry/src/components/organisms/BasketOverview/hooks/useBasketOverview.tsx:114]:
```suggestion
        startDateWindow?.seconds?.toNumber(),
```
This is erroring in the deploy preview

**Comment by mhagel** [comment]
QA findings:

OK:
- Basket Take, Put
- Create Project
- Create Batch
- Credit send, retire
- Batch table
- Activity table
- Dashboard

Not working:
- Sell Orders, BUT: I discovered this may be an issue with redwood not having REGEN (or anything) as [allowed currency](http://redwood.regen.network:1317/regen/ecocredit/marketplace/v1/allowed-denoms)
- NCT page is blowing up, probably just a null value
- Links leading to empty pages or 404s - again, _expected_ in this context, not a bug.

Overall looking good.

**Review by mhagel — APPROVED** [review]
Will create a bug for basket/uNCT page, but this is not featured in PROD, so not a blocker.
Sell Order failure is probably due to lack of allowed currencies.
Overall, seems GTG.

**Review by S4mmyb — APPROVED** [review]

**Comment by mhagel** [comment]
@mergifyio backport redwood hambach

**Pull request merged**